etc/s3-iam.cfg.in

```diff
@@ -0,0 +1,96 @@
+[DEFAULT]
+bind_port = 5000
+workers = 1
+user = %USER%
+log_facility = LOG_LOCAL0
+log_level = DEBUG
+eventlet_debug = true
+sds_default_account = AUTH_demo
+
+[pipeline:main]
+pipeline = catch_errors proxy-logging cache iam s3api tempauth proxy-logging copy slo versioned_writes symlink proxy-server
+
+[app:proxy-server]
+use = egg:swift#oio_proxy
+allow_account_management = true
+account_autocreate = true
+sds_namespace = OPENIO
+sds_proxy_url = http://127.0.0.1:6000
+log_name = OIO,NS,oioswift,1
+sds_connection_timeout=5
+sds_read_timeout=20
+sds_write_timeout=20
+delete_slo_parts=true
+
+[filter:gatekeeper]
+use = egg:swift#gatekeeper
+
+[filter:proxy-logging]
+use = egg:swift#proxy_logging
+
+[filter:catch_errors]
+use = egg:swift#catch_errors
+
+[filter:ratelimit]
+use = egg:swift#ratelimit
+
+[filter:healthcheck]
+use = egg:swift#healthcheck
+
+[filter:cache]
+use = egg:swift#memcache
+memcache_servers = 127.0.0.1:11211
+
+[filter:slo]
+use = egg:swift#slo
+
+[filter:staticweb]
+use = egg:swift#staticweb
+
+[filter:iam]
+use = egg:swift#iam
+log_level = DEBUG
+connection = file://%RULES_FILE%
+
+[filter:s3api]
+use = egg:swift#s3api
+location = RegionOne
+force_swift_request_proxy_log = true
+log_level = DEBUG
+storage_domain = localhost
+s3_acl = true
+# As of October 2020, cannot mix IAM and ACLs
```

Comment:
Is it still true with iam.py l.329 ?

Reply:
I removed the comment. `s3_acl_openbar` is not required any more (it is even not implemented).